taxonomies/categories.txt

```diff
@@ -102112,6 +103138,8 @@ ciqual_food_code:en:23415
 ciqual_food_name:en:Short crust pastry, pure butter, frozen, raw
 ciqual_food_name:fr:Pâte brisée, pur beurre, surgelée, crue
 
+# The entire section below seems to be duplicated
```

Comment:
```suggestion
```
Yeah, true. This block has <en:Pie dough and the other one has <en:Cooked puff pastry and <en:Pure butter puff pastry. Ping @moon-rabbitOFF and @aleene who may have suggestion on which parent to keep there

Reply:
Okay, I guess I will leave it for now.